content/en/docs/components/model-registry/overview.md

```diff
@@ -0,0 +1,99 @@
+
+### Use Case 2: Experimenting with Different Model Weights to Optimize Model Accuracy
+
+The _Data Scientist_ after identifying a base model, uses KubeFlow Pipelines, Katib, and other components to experiment model with alternative weights, hyperparameters, and other variations to improve the model’s performance metrics; Kubeflow Model Registry can be used to track data related to experiments and runs for comparison, reproducibility and collaboration.
+
+* Register the Base Model: Track the Base Model storage location along with hyperparameters in the Model Registry. 
+* Track Experiments/Runs: With Kubeflow pipelines or using the Kubeflow Notebooks, track every variation of the hyper-parameters along with any configuration in that specific Experiment. With each run the different parameters can be tracked in the Model Registry.
+* Track and compare performance: with each run view key metrics like accuracy, recall, and precision. This helps the Data Scientist identify the best-performing run/experiment for deployment.
+* Reproducibility: if needed, the data tracked in Model Registry can be replayed to perform again the experiment/run to reproduce the models.
+* Collaborate: Share models and experiment details with the _MLOps Engineer_ for deployment preparation. This ensures a seamless transition from training to production.
+
+### Use Case 3: Model Deployment
+
+The MLOps Engineer uses Kubeflow Model Registry to locate the most recent version for a given model, verify it is approved for deployment, understand model format, architecture, hyperparameters, and performance metrics to configure the serving environment; once deployed, Model Registry is used to continue monitoring and track deployed models for performance and mitigate drift.
+
+* Retrieve the latest model version: Easily access the model version approved for deployment.
+* Access model metadata: Understand the model's architecture, hyperparameters, and performance metrics. This helps the MLOps engineer to configure the deployment environment and monitor performance after deployment.
+* Manage serving configurations: Define how the model will be served to production applications and set up necessary resources.
+* Track model deployments: Monitor the deployed model's performance and track its health over time. This allows the MLOps Engineer to identify potential issues and take corrective actions.
+
+### Use Case 4: Monitoring and Governance
+
+The Business Analyst uses Kubeflow Model Registry to audit deployed models, monitor model performance, track key metrics and identify when model is drifting or needs re-training; capabilities of model lineage enable identifying all related artifacts such as training which was used or the original training data.
+
+* View model performance metrics: Track key metrics in real-time to understand how the model is performing in production.
```

Comment:
I this really something that the MR should do? This capability seems like part of an observability tool outside of the MR scope

Reply:
thanks for spotting, the idea is for Model Registry to be "single pane of glass" but not substituting to existing/appropriate tools. 

Implemented with 3ee7e42e7fc2edfa8c50ba390306212a4a3532e8